website/translated_docs/ar/learn-consensus.md

```diff
@@ -6,41 +6,51 @@ sidebar_label: Polkadot Consensus
 
 ## What are PoW and PoS?
 
 Proof of Work (PoW) and Proof of Stake (PoS) have been used as short hand to refer to consensus mechanisms of blockchains, but that does not capture the full picture. PoW is the method for agreeing on a block author and part of the fuller [Nakamoto consensus](#nakamoto-consensus) that also encompasses a chain selection algorithm (longest chain rule in Bitcoin). Similarly, PoS is a set of rules for selecting the validator set and does not specify a chain selection rule or how a chain might reach finality. PoS algorithms have traditionally been paired with an algorithm for coming to Byzantine agreement between nodes. For example, [Tendermint](learn-comparisons-cosmos) is a practical Byzantine fault tolerant algorithm which uses PoS as its validator set selection method.
 
-## Probable vs. provable finality
+## Why not Proof of Work?
 
-A pure Nakamoto consensus blockchain which runs PoW is only able to achieve the notion of *probable finality* and reach *eventual consensus*. Probable finality means that under some assumptions about the network and participants, if we see a few blocks building on a given block, we can estimate the probability that it is final. Eventual consensus means that at some point in the future, all nodes will agree on the truthfulness of one set of data. This eventual consensus may take a long time and will not be able to be determined how long it will take ahead of time. However, finality gadgets such as GRANDPA or Ethereum's Casper FFG are designed to give stronger and quicker guarantees on the finality of blocks - specifically, that they can never be reverted after some process of byzantine agreements has taken place. The notion of irreversible consensus is known as *provable finality.*
+Although simple and effective in coming to a decentralized consensus, proof of work consensus consumes an incredible amount of energy, has no economic or provable finality, and has no effective strategy in resisting cartels.
+
+## Probabilistic vs. provable finality
+
+A pure Nakamoto consensus blockchain which runs PoW is only able to achieve the notion of _probabilistic finality_ and reach _eventual consensus_. Probabilistic finality means that under some assumptions about the network and participants, if we see a few blocks building on a given block, we can estimate the probability that it is final.  Eventual consensus means that at some point in the future, all nodes will agree on the truthfulness of one set of data.  This eventual consensus may take a long time and will not be able to be determined how long it will take ahead of time.  However, finality gadgets such as GRANDPA or Ethereum's Casper FFG are designed to give stronger and quicker guarantees on the finality of blocks - specifically, that they can never be reverted after some process of byzantine agreements has taken place. The notion of irreversible consensus is known as _provable finality._
```

Comment:
Byzantine should be capitalized

```suggestion
A pure Nakamoto consensus blockchain which runs PoW is only able to achieve the notion of _probabilistic finality_ and reach _eventual consensus_. Probabilistic finality means that under some assumptions about the network and participants, if we see a few blocks building on a given block, we can estimate the probability that it is final.  Eventual consensus means that at some point in the future, all nodes will agree on the truthfulness of one set of data.  This eventual consensus may take a long time and will not be able to be determined how long it will take ahead of time.  However, finality gadgets such as GRANDPA or Ethereum's Casper FFG are designed to give stronger and quicker guarantees on the finality of blocks - specifically, that they can never be reverted after some process of Byzantine agreements has taken place. The notion of irreversible consensus is known as _provable finality._
```